src/domain/Config.ts

```diff
@@ -0,0 +1,10 @@
+import ConnectionConfig from './ConnectionConfig';
+
+/**
+ * Interface for sync-db connection file (connections.sync-db.json).
+ */
+interface Config {
```

Comment:
Rename to `SyncDbConfig`. There are too many configuration that we are playing with. Let's make it obvious.